test/Microsoft.NET.Build.Tests/GivenThatWeWantToVerifyProjectReferenceCompat.cs

```diff
@@ -0,0 +1,213 @@
+        [InlineData("netstandard1.5", "netstandard1.2", true)]
+        [InlineData("netstandard1.5", "netstandard1.3", true)]
+        [InlineData("netstandard1.5", "netstandard1.4", true)]
+        [InlineData("netstandard1.5", "netstandard1.5", true)]
+
+        [InlineData("netstandard1.6", "netstandard1.0", true)]
+        [InlineData("netstandard1.6", "netstandard1.1", true)]
+        [InlineData("netstandard1.6", "netstandard1.2", true)]
+        [InlineData("netstandard1.6", "netstandard1.3", true)]
+        [InlineData("netstandard1.6", "netstandard1.4", true)]
+        [InlineData("netstandard1.6", "netstandard1.5", true)]
+        [InlineData("netstandard1.6", "netstandard1.6", true)]
+
+        [InlineData("netstandard2.0", "netstandard1.0", true)]
+        [InlineData("netstandard2.0", "netstandard1.1", true)]
+        [InlineData("netstandard2.0", "netstandard1.2", true)]
+        [InlineData("netstandard2.0", "netstandard1.3", true)]
+        [InlineData("netstandard2.0", "netstandard1.4", true)]
+        [InlineData("netstandard2.0", "netstandard1.5", true)]
+        [InlineData("netstandard2.0", "netstandard1.6", true)]
+        [InlineData("netstandard2.0", "netstandard2.0", true)]
+
+        [InlineData("netcoreapp1.0", "netstandard1.0", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.1", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.2", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.3", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.4", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.5", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.6", true)]
+        [InlineData("netcoreapp1.0", "netcoreapp1.0", true)]
+
+        [InlineData("netcoreapp1.1", "netstandard1.0", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.1", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.2", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.3", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.4", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.5", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.6", true)]
+        [InlineData("netcoreapp1.1", "netcoreapp1.0", true)]
+        [InlineData("netcoreapp1.1", "netcoreapp1.1", true)]
+
+        [InlineData("netcoreapp2.0", "netstandard1.0", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.1", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.2", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.3", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.4", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.5", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.6", true)]
+        [InlineData("netcoreapp2.0", "netstandard2.0", true)]
+        [InlineData("netcoreapp2.0", "netcoreapp1.0", true)]
+        [InlineData("netcoreapp2.0", "netcoreapp1.1", true)]
+        [InlineData("netcoreapp2.0", "netcoreapp2.0", true)]
```

Comment:
Doing some looking at the Jenkins runs, it looks like these tests are doubling the job time in Jenkins.

Runs without your change take between 7-8 minutes:
https://ci.dot.net/job/dotnet_sdk/job/master/job/release_windows_nt_prtest/217/
https://ci.dot.net/job/dotnet_sdk/job/master/job/release_windows_nt_prtest/218/

Your runs are taking 16 minutes:
https://ci.dot.net/job/dotnet_sdk/job/master/job/release_windows_nt_prtest/223/

Are we getting that much value out of these tests?